scripts/ipintutil

```diff
@@ -48,8 +48,8 @@ def get_bgp_peer():
     data = config_db.get_table('BGP_NEIGHBOR')
 
     for neighbor_ip in data.keys():
-        local_addr = data[neighbor_ip]['local_addr']
-        neighbor_name = data[neighbor_ip]['name']
+        local_addr = data[neighbor_ip].get('local_addr', "")
```

Comment:
might be better to just put the whole block inside the for loop inside:
```python
try:
    ...
except KeyError:
  pass
```

Your current solution adds a blank entry to `bgp_peer` which doesn't do anything for us...

Should probably also put a comment that this get_bgp_peer is for retrieving manually specified BGP peers only, not BGP Unnumbered peers (like when specifying an interface)